lib/formtools/renderers-cell.js

```diff
@@ -8,7 +8,7 @@ var dateRenderer = module.exports.date = function dateRenderer (date, record, fi
     return callback(null, moment(date).format(linz.get('date format')));
 };
 
-var dateTimeRenderer = module.exports.dateTime = function dateTimeRenderer (date, record, fieldName, model, callback) {
```

Comment:
we renamed few of these variable names in renderers-cell.js to match them with field type returned by `model.linz.formtools.form[field]`.